shpc/main/modules/base.py

```diff
@@ -470,7 +473,14 @@ def install(
 
         # Get the template based on the module and container type
         template = self.template.load(self.templatefile)
-        module_path = os.path.join(module.module_dir, self.modulefile)
+        if self.settings.version_naming:
+            self._modulefile = name.split(":")[1]
```

Comment:
What is `self._modulefile` used for? I only see it in this small set of statements. It might make sense to just use `name.split(":")[1]` in the one place it is needed and not save a private class variable.